### DISCUSSION TITLE
【命令分享的步骤】【复制xml】小伙伴们可以按照这个方式分享自己的命令

### DISCUSSION THREAD

**Discussion by ttbamov** [Chaoses-Ib/DirectoryOpus#3] [分享]
### **1、工具栏右键→自定义→复制命令（得到xml内容）**
![image](https://github.com/Chaoses-Ib/DirectoryOpus/assets/128291620/d1760e76-c46e-49e2-9336-ee51bb347f2a)
### **2、在txt文本里面编辑好内容，如下图所示**
![image](https://github.com/Chaoses-Ib/DirectoryOpus/assets/128291620/891d4c00-d675-4b54-8dcf-992a9e815b90)
**演示动作命令**

```js
function OnClick(clickData)
{
var cmd = clickData.func.command;
cmd.deselect = false;
var focesIten = clickData.func.sourcetab.GetFocusItem;
var name = focesIten.name_stem_m.replace(/\s*(.*/,"")
cmd.AddLine('@set name={dlgstringS|请输入新建文件夹名称|"' + name + '"}');
cmd.AddLine('Copy MOVE HERE CREATEFOLDER="{$name}"');
cmd.Run;
}
```

【`】这个符号是不方便打可以复制过去

### **3、描述命令内容并发布**
这样就就可以和小伙伴们一起分享Directory Opus配置知识了

**Comment by Chaoses-Ib** [comment]
![image](https://github.com/Chaoses-Ib/DirectoryOpus/assets/50246090/11dfd0cc-acfb-44eb-8ffc-11100e67578f)

<kbd>`</kbd> 在键盘左上角，或者也可以在 GitHub 中选中代码后点这个代码块按钮

**Comment by Chaoses-Ib** [comment]
![image](https://github.com/Chaoses-Ib/DirectoryOpus/assets/50246090/b5a6aa5b-1f0c-444e-a146-25df0a4bf74c)

GitHub 上编辑时可以预览
